src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -203,6 +203,8 @@ internal int CurrentPasteOffset
         private DateTime lastSaved;
         private string author = "None provided";
         private string description;
+        private string thumbnail;
```

Comment:
before this gets merged- a quick thought, do all of these new properties make sense on both home workspaces and custom nodes workspaces?

Reply:
Tbh I hadn't considered that this also affected the CN workspace. Most of these probably don't make sense on the CN, maybe the `Author` is okay, but not sure i see the need for the others.

@nate-peters @saintentropy, any thoughts?

Reply:
Also, just noticed that its the same with the new `ExtensionWorkspaceData` not sure that makes sense in the CN workspace either?